policies/cla.md

```diff
@@ -0,0 +1,63 @@
+# Contributor Agreements
+
+Every non-trivial contribution needs to be covered by a signed
+Contributor License Agreement (CLA) from all original authors. We have
+modelled our policy based on the practice of [the Apache Software
+Foundation](https://www.apache.org). You can see their CLA policy
+[here](https://www.apache.org/licenses/#clas). Our policy is:
+
+OpenSSL requires that all non-trivial contributors of ideas, code, or
+documentation complete, sign, and submit (via postal mail, fax or email)
+an [Individual Contributor License Agreement (ICLA)](openssl_icla.pdf).
+The purpose of this agreement is to clearly define the terms under which
+intellectual property has been contributed to OpenSSL and thereby allow
+us to defend the project should there be a legal dispute regarding the
+software at some future time.
+
+A submission is trivial if it is considered trivial under copyright law.
+Since we are not lawyers, we place the bar for trivial contributions
+very high. For example: corrections of grammatical or typographical
+errors (including misspelled function names in manual pages), simple
+whitespace changes and in some cases one-line bugfixes might be accepted
+as trivial without requiring a CLA.
+
+In practice, it is required that the author (in the git commit message)
+and all approving team members (in the pull request thread) agree that a
+change is trivial. The author has to add "CLA: trivial" in the commit
+message separated by an empty line from the rest of the message. The
+reviewers will normally post a statement to the effect of "I agree that
+it is a trivial change."
+
+*When filling in the CLA, please make sure that the email address
+matches the one that you use for the "Author" in your git commits.
+List multiple email addresses if necessary.*
+
+For a corporation that has assigned employees to work on OpenSSL, a
+[Corporate Contributor License Agreement (CCLA)](openssl_ccla.pdf) is
+available for contributing intellectual property via the corporation,
+that may have been assigned as part of an employment agreement. Note
+that a Corporate CLA does not remove the need for every developer to
+sign their own ICLA as an individual.
+
+If you have not already done so, please complete and sign a printout of
+the above ICLA (and CCLA if necessary), then scan and email a pdf file
+of the Agreement(s) to <legal@opensslfoundation.org>.
+
+If you prefer snail mail, send an original signed Agreement to the
+
+OpenSSL Software Foundation\
```

Comment:
Quote block for this?  Or at least indent?

Reply:
Also EOL \\'s should go

Reply:
> Also EOL \'s should go

No, they are line breaks.